### DISCUSSION TITLE
DBeaver web based

### DISCUSSION THREAD

**Discussion by ashokeir** [dbeaver/dbeaver#23323] [General]
Hello everyone!
I'm wondering if anyone knows if there's a web-based version of DBeaver available? The new company I'm with doesn't allow me to install any software on my laptop, so I'm looking for an alternative solution.
Thank you in advance for any assistance you can provide!
If so can you share the URL - I hope it is free...

**Comment by mkgrgis** [comment]
https://dbeaver.com/docs/cloudbeaver/
